lib/xlsx/xform/sheet/sheet-format-properties-xform.js

```diff
@@ -13,6 +13,7 @@ class SheetFormatPropertiesXform extends BaseXform {
         outlineLevelRow: model.outlineLevelRow,
         outlineLevelCol: model.outlineLevelCol,
         'x14ac:dyDescent': model.dyDescent,
+        customHeight: model.defaultRowHeight === 0 ? '0' : '1',
```

Comment:
Could you fix the unit tests that have failed due to this change

Reply:
Yes